Makefile

```
@@ -466,8 +466,9 @@ ifeq ($(PCH), 1)
   endif
 endif
 
+DEPFLAGS=-MMD -MP
```

Variable assignments in the rest of Makefile usually have spaces.
```suggestion
DEPFLAGS = -MMD -MP
```